modelhub/modelhub/modelhub.py

```diff
@@ -69,31 +72,39 @@ def conversion_events(self):
         """
         return self._conversion_events
 
-    def _check_data_is_objectiv_data(self, data):
-        if data.index_dtypes != {'event_id': 'uuid'}:
-            raise ValueError(f"not right index {data.index_dtypes}")
-
-        required_columns = {
-            'day': 'date',
-            'moment': 'timestamp',
-            'user_id': 'uuid',
-            'global_contexts': 'objectiv_global_context',
-            'location_stack': 'objectiv_location_stack',
-            'event_type': 'string',
-            'stack_event_types': 'json',
-            'session_id': 'int64',
-            'session_hit_number': 'int64'
-        }
-
-        if not (required_columns.items() <= data.dtypes.items()):
-            raise ValueError(f"not right columns in DataFrame {data.dtypes.items()}"
-                             f"should be {required_columns.items()}")
-
-    def get_objectiv_dataframe(self,
-                               db_url: str = None,
-                               table_name: str = 'data',
-                               start_date: str = None,
-                               end_date: str = None):
+    @staticmethod
+    def _check_data_is_objectiv_data(data: bach.DataFrame):
+        """
+        Validates that provided DataFrame is a representation of an objectiv dataframe.
+        """
+        from modelhub.stack.util import check_objectiv_dataframe
+        check_objectiv_dataframe(data, check_index=True, check_dtypes=True, with_md_dtypes=True)
+
+    @staticmethod
+    def _get_db_engine(db_url: Optional[str], bq_credentials_path: Optional[str] = None) -> Engine:
+        """
+        returns db_connection based on db_url.
+        If db_url is for BigQuery, bq_credentials_path must be provided.
+        """
+        if db_url and re.match(r'^bigquery://.+', db_url):
+            if not bq_credentials_path:
+                raise ValueError('BigQuery credentials path is required for engine creation.')
+
+            return create_engine(db_url, credentials_path=bq_credentials_path)
+
+        import os
+        db_url = db_url or os.environ.get('DSN', 'postgresql://objectiv:@localhost:5432/objectiv')
+        return create_engine(db_url)
+
+    def get_objectiv_dataframe(
+        self,
+        db_url: str,
+        table_name: str,
```

Comment:
I think I removed the default value
```suggestion
        table_name: str = 'data',
```